### PR TITLE
Laurentiu oncescu meet me

### PR DESCRIPTION
**Pull request by biggicode**

### CODE REVIEW

File: src/_component/contact/contact.pug [hunk 1]
```diff
@@ -0,0 +1,43 @@
+- var {sectionTitle}=data['contact']
```
Comment (marianzburlea) [inline src/_component/contact/contact.pug:1]: add spaces to increase readablity `{ sectionTitle }`

Comment (marianzburlea) [reply src/_component/contact/contact.pug:1]: also for the = sign:
` = `

File: src/_component/contact/contact.pug [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+include ../section-title/section-title
+
+.column(span="row")
+  .contact__profile-picture
+.column(span="row" height="10")
```
Comment (marianzburlea) [inline src/_component/contact/contact.pug:6]: it is ideal to leave an empty line to read quicker

File: src/_component/contact/contact.pug [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    type="radio"
+    name="contact__label"
+    id=`contact__label--${cheie}`
+  )
+  .column(span="1")
+    label(for=`contact__label--${cheie}`).contact__icon
+      div(class=`icon__${iconita.icon}`)
+
+each iconita, cheie in data.contact.iconsList
+  .column(span="row" about-me-box=cheie+1)
```
Comment (marianzburlea) [inline src/_component/contact/contact.pug:18]: add spaces:
`cheie + 1`

File: src/_component/contact/contact.pug [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    .about-me-box__with-arrow
+      .about-me-box__title
+        .about-me-box__title-text=iconita.title
+        .about-me-box__cross
+          input.contact__radio(
+            type="radio"
+            name="contact__label"
+            id=`contact__label--remove`
+          )
+          label(for=`contact__label--remove`).contact__icon
```
Comment (marianzburlea) [inline src/_component/contact/contact.pug:28]: for consistency:
```
label.contact__icon(for=`contact__label--remove`)
```

File: src/_component/contact/contact.pug [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            div(class=`icon__cross`)
+      .about-me-box__content
+        p.about-me-box_paragraph=iconita.description
+      .about-me-box__end
+        a(
+          href=iconita.link
+          target="_blank"
+        ).btn-download-blue
```
Comment (marianzburlea) [inline src/_component/contact/contact.pug:36]: move the class before the attributes

File: src/_component/contact/contact.scss [hunk 6]
```diff
@@ -0,0 +1,124 @@
+.contact__radio:checked + .column .contact__icon {
+  background-color: #d52027;
+
+}
+
+@for $pere from 1 to 11 {
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:6]: you can use `to` or `through`

File: src/_component/contact/contact.scss [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    justify-content: center;
+    align-items: center;
+   }
+}
+
+.contact__profile-picture {
+  padding-top: 100%;
+  background-color: white;
+  position: relative;
+  box-shadow:0 2px 6px rgba(0, 0, 0, 0.25);
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:50]: add a space before 0

File: src/_component/contact/contact.scss [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+  border-radius: 8px;
+  transform: rotate(-2.5deg);
+  
+  &::after {
+    content: '';
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:55]: always remember to add content with at least '' as a value for before and after

File: src/_component/contact/contact.scss [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+.about-me-box__with-arrow {
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.25);
+  border-radius: 6px;
+  background-color: #fff;
+  position: relative;
+  margin-top: 1rem;
+  
+  
+  &::after {
+    content: "";
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:76]: `content: '';`

File: src/_component/contact/contact.scss [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+  display: flex;
+}
+
+.about-me-box__title-text {
+  flex-grow: 1;
+  line-height: 40px;
+  font-weight: 500;
+  font-size: 22px;
+  text-transform: capitalize;
+
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:100]: remove empty line

File: src/_component/contact/contact.scss [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+.about-me-box__content {
+  border-top: 2px solid #F5F5F5;
+  border-bottom: 2px solid #F5F5F5;
+}
+
+.about-me-box_paragraph {
+  font-size: 14px;
+  line-height: 30px;
+}
+
```
Comment (marianzburlea) [inline src/_component/contact/contact.scss:122]: keep only one line

File: src/_data/contact.json [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    {
+      "icon":"twitter",
+      "title": "Twitter",
+      "description": "lafoqmfomaofmaomfom",
+      "label": "codetap.academy",
+      "link": "https://codetap.academy"
+    }
+  ]
+}
```
Comment (marianzburlea) [inline src/_data/contact.json:76]: add final line

File: src/_data/welcome-data.json [hunk 13]
```diff
@@ -7,5 +7,5 @@
     "I am a shy person which once I manage to attach myself to someone or a group of people I can't stop talking.Once someone becames a part of my life I like to help them and see them succed.Happy people makes me happy too.I want to see more happy people.In the end this what'll remain with,all the interactions and emotions with people that we met.",
     "I like working in web development because web offers me the posibility to work on projects that can be seen by a lot of people around the globe.While I work on a project I can see my work taking shape every minute,for me this is a very important thing because this makes me feel very happy."
   ],
-  "btnDownloadText": "Click to get my"
+  "btnDownloadText": "Click to get my CV"
 }
```
Comment (marianzburlea) [inline src/_data/welcome-data.json:11]: add final line

File: src/style.scss [hunk 14]
```diff
@@ -1,4 +1,6 @@
 @import '_component/layout/roboto-font';
 @import '_component/layout/layout';
+@import '_component/section-title/section-title';
 @import '_component/welcome-section/welcome-section';
+@import '_component/contact/contact';
 @import '_asset/font/style';
```
Comment (marianzburlea) [inline src/style.scss:6]: eol

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Review by gratielazburlea — APPROVED** [review]

**Review by ValentinaVanu — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**